### ISSUE TITLE
Azure OpenAI: Unknwon parametr: `stream_options`

### ISSUE THREAD

**Issue by pstanisl** [BoltAI/BoltAI#135]
Azure OpenAI rejecting requests after the update with the message

```
Sorry, your request has been rejected. Here is the error message: Unknwon parameter: 'stream_options'.
```

![CleanShot 2024-07-29 at 10 27 36@2x](https://github.com/user-attachments/assets/ab9638b6-4ec2-4b0d-9297-729f7d846118)

BoltAI version v1.17.1 (build 92)

![CleanShot 2024-07-29 at 10 33 44@2x](https://github.com/user-attachments/assets/7820ee3c-24d6-4e24-afb8-3962358e0df3)

**Comment by longseespace** [comment]
Ah sorry for the issue. Going to fix soon.

**Comment by longseespace** [comment]
I've fixed this in v1.17.2. Sorry for the issue.

**Comment by pstanisl** [comment]
Thank you for super fast response and for the great application.
